### ISSUE TITLE
[3.x] Composable not auto-imported

### ISSUE THREAD

**Issue by rcerljenko** [johannschopplich/nuxt-gtag#90]
### Environment

nuxt 3.13

### Reproduction

![image](https://github.com/user-attachments/assets/3fe6d5ba-a6de-4cd9-b21c-a914fa3544b2)

### Describe the bug

It seems that `useGtag()` composable is not auto-imported in layout file

**Comment by johannschopplich** [comment]
Thanks for the report! Please follow the issue convention and provide a reproducible example. You can use one of the templates below to create a minimal reproduction:
👉 https://stackblitz.com/github/nuxt/starter/tree/v3-stackblitz
👉 https://codesandbox.io/p/github/nuxt/starter/v3-codesandbox

Thanks in advance!

**Comment by rcerljenko** [comment]
Ok i've got it... it seems that now when `enabled` flag is `false`, nuxt doesn't import the composable anymore

**Comment by johannschopplich** [comment]
You're right. With v3, the `enabled` module option now controls whether the module should be active for your project at all. If the module is disabled, the composables won't be importable.

**Comment by ptdev** [comment]
Hi,

Sorry to add to a closed ticket, but we also got hit by a similar issue.

As per the documentation, we started using the following:

```
gtag: {
    enabled: process.env.NODE_ENV === 'production',
    id: 'G-XXXXXXXXXX'
}
```

Which in fact disables the module when in development.

But then everywhere that we're calling useTrackEvent, we're now getting console errors that the function is not defined. 

On some cases, for example if we're calling it inside a try/catch block, then the catch get's triggered when on development.

It would be useful to have a way to disable the sending of analytics when not in production without disabling the importing of the composables which forces us to add an if check everytime we call the composable.
Maybe have the composables just return void when disabled or something (?)

Just a thought.

Thank you, cheers
